CodeEditModules/Modules/StatusBar/src/Model/StatusBarModel.swift

```diff
@@ -39,17 +64,25 @@ public class StatusBarModel: ObservableObject {
     public var currentCol: Int = 1 // Implementation missing
 
     /// Returns true when the drawer is visible
-    @Published
+    @AppStorage(StatusBarStorageKey.isExpanded.rawValue)
     public var isExpanded: Bool = false
 
+    /// Returns true when the drawer is visible
+    @AppStorage(StatusBarStorageKey.isMaximized.rawValue)
+    public var isMaximized: Bool = false
+
     /// The current height of the drawer. Zero if hidden
-    @Published
+    @AppStorage(StatusBarStorageKey.currentHeight.rawValue)
```

Comment:
I don't think AppStorage is a good way to go. What if a user has more than one workspace open? Then it would change in all workspaces.